app/frontend/test/unit/specs/aquifers/components/__snapshots__/View.spec.js.snap

```diff
@@ -700,9 +700,9 @@ exports[`View Component View mode matches the snapshot 1`] = `
       <div
         class="col"
       >
-        <h5>
+        <h4>
```

Comment:
Why h5 -> h4? Is it a visual thing? Accessibility? Or actually relating to the tests?

Reply:
2 reasons:
1.  I needed some way to target the page header, as part of the test (standard practice thus far to ensure you are on the correct page).
2.  All of the other pages use an 'h4' for the headers.  The aquifer pages were the only ones that deviated, and as a result, had multiple h5 tags, which made the (1) difficult.